frontend/src/js/controllers/authCtrl.js

```diff
@@ -115,9 +115,10 @@
                                 }
 
                             } catch (error) {
-                                console.log(error);
+                                $rootScope.notify("error",error);
```

Comment:
Can you please replace this line with `$rootScope.notify(error);` ?

Reply:
Ready.

Reply:
Can you explain what is notify "success" argument? I don't can find them in from where it are injected.